package.json

```diff
@@ -79,11 +77,11 @@
     "sinon": "^9.2.4",
     "snyk": "^1.836.0",
     "sort-package-json": "^1.53.1",
-    "ts-node": "^9.1.1",
+    "ts-node": "^10.9.2",
```

Comment:
Do these upgrade consider breaking change?

Reply:
It shouldn't as `ts-node` and `typescript` are meant for development/building and not for the compiled SDK itself